docs/mcr-endpoints-guidance.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Guidance for using the official Microsoft Container Registry (MCR) endpoints
+description: This articles provides information about the official endpoints of Microsoft Container Registry (MCR) and provides guidance how to use the REST and CDN endpoints to pull official Microsoft artifacts around the world.
+ms.topic: guidance
+ms.date: 02/03/2021
+ms.author: memladen
+author: toddysm
+ms.custom:
+---
+
+# Guidance for using the official Microsoft Container Registry (MCR) endpoints
+
+Microsoft Container Registry (MCR) is an implementation of the [OCI Distribution Specification][oci-spec] which delivers [artifacts][oci-artifacts], such as container images, helm charts and other assets distributed by Microsoft and Azure. The Distribution Spec linked above defines two endpoints:
+
+- **Registry Endpoint**: providing content discovery. This is also referred to as *REST Endpoint* and it is the endpoint users are most familiar with when pulling an image; for example: `docker pull mcr.microsoft.com/windows/servercore:1909`. The registry endpoint is load balanced across multiple worldwide regions, providing reliable and consistent access to the MCR artifacts catalog.
+- **Data Endpoint**: providing content delivery. As a registry client discovers the content required, it negotiates a set of content urls, pulling from the data endpoint. To provide reliable delivery of content, the data endpoint is backed by regional CDN endpoints.
+
+## Official FQDNs for MCR
+
+To access MCR, the following FQDNs should be used.
+
+| Purpose | Protocol | Target FQDN |
+| - | - | - |
+| Registry Endpoint | https | `mcr.microsoft.com` |
+| Data Endpoint | https | `*.data.mcr.microsoft.com` |
+
+> **Note:** MCR provides global coverage through Azure Traffic Manager for the registry endpoint, with regional CDNs managed by Azure Front Door for the data endpoints.
+> Over time, MCR team will continue adding regional endpoints to ensure the best performance for our customers. Check the [MCR Firewall Rules][mcr-firewall-rules] article for details how to configure your firewall rules.
```

Comment:
This link didn't work for me. Is it supposed to work after the article is published?

Reply:
Fixed